src/style.css

```diff
@@ -1,3 +1,115 @@
 body {
-    background-color: bisque;
-}
\ No newline at end of file
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  box-sizing: border-box;
+}
+
+main {
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 2px 8px rgb(0 0 0 / 15%);
+  border-radius: 3px;
+  width: 70%;
+}
+
+.title {
+  font-size: 0.9rem;
+  font-weight: bold;
+  padding-left: 0.9rem;
+}
+
+.app-head {
+  display: flex;
+  justify-content: space-between;
+  text-align: left;
+  color: #545862;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+}
+
+i .app-head {
+  font-size: 0.9rem;
+  padding-top: 0.8rem;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+}
+
+.info {
+  outline: none;
+  border: none;
+  padding: 20px 0 20px 20px;
+  width: 100%;
+  font-style: italic;
+  font-size: 0.9rem;
+  color: #7a7575;
+}
+
+ul {
+  width: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  display: flex;
+  list-style-type: none;
+  padding: 20px 0 20px 20px;
+  margin-left: 0;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+}
+
+i {
+  margin-right: 1rem;
+  color: #827f7f;
+}
+
+.clear-done {
+  background: transparent;
+  margin: 1rem auto;
+  padding-top: 0;
+  border: none;
+  cursor: pointer;
+  font-size: 0.9rem;
+  opacity: 0.5;
+}
+
+.clear-done:hover {
+  text-decoration: underline;
+}
+
+.clear-list {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: rgb(231, 230, 230);
+  width: 100%;
+}
+
+.list-input {
+  border: none;
+  outline: none;
+  width: 100%;
+  background-color: transparent;
+  cursor: text;
+  font-size: 0.8rem;
+  color: #7a7575;
+}
+
+.desc {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.tick {
+  margin-right: 0.5rem;
+}
```

Comment:
- [ ] Please try to reduce the `width` and size of your to-do list in order to make it similar to the project captured in the video below :arrow_down: Note that as a developer it is very important to pay attention to detail. :smile: 

#### Project capture
![68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067 (1)](https://user-images.githubusercontent.com/6615394/193027844-39fe2480-4d27-4b22-ba4c-39f53419fc79.jpeg)

#### Your todo list design
![Screenshot from 2022-09-29 12-04-59](https://user-images.githubusercontent.com/6615394/193027852-bf153e17-a70d-4bb7-92be-eb3217aa6fd9.png)